tutorials/pretraining-data-curation/red-pajama-v2-curation-tutorial.ipynb

```diff
@@ -0,0 +1,6286 @@
+    "- Slurm job directives\n",
+    "- Device type (`cpu` or `gpu`). Curator has both cpu and gpu modules. Check [here](https://docs.nvidia.com/nemo-framework/user-guide/latest/datacuration/cpuvsgpu.html) to see which modules are cpu/gpu\n",
+    "- Path to the NeMo Framework container image\n",
+    "- Path to `container-entry-point.sh` script which is responsible for launching the dask schduler and workers\n",
+    "\n",
+    "Running the script will also launch a jupyter lab session on the rank 0 node and pass the dask schduler address as an environment variable that will be used later to connect to the dask client.\n",
+    "\n",
+    "The preprocessing modules such as Add ID and Text cleaning are cpu-based so we will start a cpu dask cluster first."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "id": "5de0fe93",
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "/usr/local/lib/python3.10/dist-packages/tqdm/auto.py:21: TqdmWarning: IProgress not found. Please update jupyter and ipywidgets. See https://ipywidgets.readthedocs.io/en/stable/user_install.html\n",
+      "  from .autonotebook import tqdm as notebook_tqdm\n"
+     ]
+    }
+   ],
+   "source": [
+    "import os\n",
+    "import time\n",
+    "from dask.distributed import Client\n",
+    "import warnings\n",
+    "import dask.dataframe as dd\n",
+    "import dask_cudf\n",
+    "import cudf\n",
+    "import gzip\n",
+    "import json\n",
+    "import dask.bag as db\n",
+    "import glob\n",
+    "from dask.distributed import wait\n",
+    "import numpy as np\n",
+    "\n",
+    "from nemo_curator.datasets import DocumentDataset\n",
+    "from nemo_curator.utils.distributed_utils import (\n",
+    "    get_num_workers,\n",
+    "    read_data,\n",
+    "    write_to_disk,\n",
+    ")\n",
+    "from nemo_curator.utils.file_utils import (\n",
+    "    expand_outdir_and_mkdir, \n",
+    "    get_all_files_paths_under, \n",
+    "    separate_by_metadata,\n",
+    "    get_batched_files,\n",
+    ")\n",
+    "\n",
+    "os.environ[\"DASK_DATAFRAME__QUERY_PLANNING\"] = \"False\"\n",
+    "os.environ[\"UCX_IB_ROCE_LOCAL_SUBNET\"] = \"y\"\n",
+    "# os.environ[\"UCX_IB_TIMEOUT\"] = \"15s\"  # Increase InfiniBand timeout\n",
+    "# os.environ[\"UCX_RNDV_TIMEOUT\"] = \"15s\"  # Increase rendezvous protocol timeout\n",
+    "warnings.filterwarnings('ignore')\n",
+    "\n",
+    "base_dir = \"/lustre/fsw/portfolios/coreai/users/yayu/data.fs5\""
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "id": "8d728cab-d3ad-41b8-aae8-6e6927f7edaf",
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Num Workers = 256\n"
+     ]
+    }
+   ],
+   "source": [
+    "scheduler_address = os.getenv('SCHEDULER_ADDRESS')\n",
+    "cpu_client = Client(address=scheduler_address)\n",
```

Comment:
Can you use `from nemo_curator import get_client` for this?

Reply:
changed to use`get_client` throughout notebook